Makefile

```diff
@@ -167,15 +168,18 @@ test-single-integration:
 
 .PHONY: test-ci-unit
 test-ci-unit: test-base
+	cat $(coverfile) | egrep -v -f $(coverage_exclude) | sponge $(coverfile)
```

Comment:
Does this work actually? I think `egrep` matches the full expression whereas in the cover file the lines look like `xxx_mock.go:94,37,97.2 2 0`.

Reply:
Yep:
```
❯ cat test.out
try/something.go:13432
random_notmatch.go:1234223423
this-should-work:23423

❯ cat filters.out
try/

❯ cat test.out | egrep -v -f filters.out
random_notmatch.go:1234223423
this-should-work:23423
```

Reply:
Ah cool.